### PR TITLE
Tabbing, select all in cells, and change single click to ctrl click

### PR DESCRIPTION
**Pull request by cfonseca88**
Thanks for this great npm for quill version 1. This module seems to work better than most of the Quill 2.0 table modules even, especially with the multi-line in cell!

I have added some minor optional key bindings which seem to fit nicely with what you have currently in keyboardHandler. I have added a tabbing bind so the user can quickly enter data move to the next cell or previous cell.

I have also added a custom optional select all for the cell alone, for quick delete, or edit of text. This works nicely with your implementation of multi-line per cell which has been appreciated.

Lastly, I added a different approach for cell select (previous single click). I appreciated that it was a separate function for click and double click but found it much easier as a ctrl click for selecting the cell. Regular click as the entry to cell to begin typing, etc.

### CODE REVIEW

File: src/demo.js [hunk 1]
```diff
@@ -59,6 +60,35 @@ const quill = new Quill(document.getElementById('quillContainer'), {
...
+              let delta = new Delta().retain(range.index)
+                                     .delete(range.length)
+                                     .insert('\t');
+              this.quill.updateContents(delta, Quill.sources.USER);
+              this.quill.history.cutoff();
+              this.quill.setSelection(range.index + 1, Quill.sources.SILENT);
+            }
+          }
+        },
+        customShiftTab: {
```
Comment (dclement8) [inline src/demo.js]:
```suggestion
        shiftTab: {
```

File: src/demo.js [hunk 2]
```diff
@@ -59,6 +60,35 @@ const quill = new Quill(document.getElementById('quillContainer'), {
...
+          key: 'tab',
+          shiftKey: true,
+          handler: function (range, keycontext) {
+            return TableModule.keyboardHandler(this.quill, 'shiftTab', range, keycontext);
+          }
+        },
+        customSelectAll: {
```
Comment (dclement8) [inline src/demo.js]:
```suggestion
        selectAll: {
```

**Review by dclement8 — COMMENTED** [review]
Hi, thank you very much for your contribution!
I tested and everything works. I have made some suggestions on your PR.

Another suggestion: for cell selection, I think we should make it configurable:
- with the CTRL key (and single click for editing the cell) => your implementation
- single click (and double click for editing the cell) => previous implementation

https://quilljs.com/guides/building-a-custom-module/#using-options
CTRL key by default except if we pass "holdCtrlKeyForCellSelection: false" or "cellSelectionOnClick" option (or something similar, I'm not sure what the most appropriate name would be)

**Comment by cfonseca88** [comment]
Thanks for the suggestion! I went ahead and added that with your keybind suggestion aswell. I have made it so it is set to cellSelection on by default to match the original demo. If the user configs to table:true in modules it will be the default behavior aswell or unless otherwise optioned

**Pull request merged**